framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2379,7 +2377,7 @@ public boolean isFromByteCode(Element element) {
      * name equals the passed annotationName if one exists, null otherwise. This
      * is the private implementation of the same-named, public method.
```

Comment:
You made this method protected, but didn't change this javadoc.  Also, why did you make it protected?